Assets/Scripts/Game/GameInformation.cs

```diff
@@ -52,19 +52,8 @@ private void EvaluateGameState()
                 // if time over, evaluate depending on integrity value
                 currentGameState = integrity.GetCurrentIntegrity() > IntegrityLostThreshold ? GameState.GameWon : GameState.GameLost;
 
-                switch (currentGameState)
-                {
-                    case GameState.GameLost:
-                        timelineManager.PlayEndSceneLose();
-                        break;
-                    case GameState.GameWon:
-                        timelineManager.PlayEndSceneWin();
-                        break;
-                    case GameState.Ongoing:
-                        break;
-                    default:
-                        throw new ArgumentOutOfRangeException();
-                }
+                // plays the timeline at the end of the game, when the player lost
+                if(currentGameState == GameState.GameLost) timelineManager.PlayEndSceneLose();
```

Comment:
The `GameInformation` script is not intended to change the game state or execute any actions on game components, but should only observe the game state and evaluate changes to notify observers.
A more separated and cleaner way would be to add a `GameInformation`-reference to the TimelineManager-Script and listen to the OnGameStateChanged event.
E.g:
```csharp
gameInformation.OnGameStateChanged += state => {
    if(state == GameState.GameLost) PlayEndSceneLose();
};
```